test/run-shared-integration-tests

```diff
@@ -6,7 +6,7 @@ set -eo pipefail
 
 declare project_root="${0%/*}/.."
 declare ringpop_common_dir="${0%/*}/ringpop-common"
-declare ringpop_common_branch="master"
+declare ringpop_common_branch="menno/node-4-compat"
```

Comment:
Is that branch really necessary? Looks like the only change is tchannel:

```
motiejus ~/Uber/ringpop-common $ git diff $(git merge-base origin/master origin/menno/node-4-compat) origin/menno/node-4-compat
diff --git a/test/package.json b/test/package.json
index bdb328d..05288d2 100644
--- a/test/package.json
+++ b/test/package.json
@@ -20,7 +20,7 @@
     "tap-parser": "^1.2.2",
     "tape": "^4.2.0",
     "tape-catch": "^1.0.5",
-    "tchannel": "^2.4.6",
+    "tchannel": "^3.6.13",
     "underscore": "^1.8.3"
   }
 }
```

Should we update the tchannel on master in integration tests instead?